src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SemanticTokensEditsDiffer.cs

```diff
@@ -67,37 +67,59 @@ private Container<SemanticTokensEdit> ProcessEdits(IReadOnlyList<DiffEdit> diffs
                         }
                         else
                         {
-                            results.Add(new SemanticTokensEdit
+                            razorResults.Add(new RazorSemanticTokensEdit
                             {
                                 Start = diff.Position,
-                                Data = ImmutableArray<int>.Empty,
                                 DeleteCount = 1,
                             });
                         }
                         break;
                     case DiffEdit.Type.Insert:
                         if (current != null &&
-                            current.Data.HasValue &&
-                            current.Data.Value.Any() &&
+                            current.Data != null &&
+                            current.Data.Any() &&
```

Comment:
Just trying to understand; why is this preferred to what we had before?

Reply:
We don't have access to .HasValue and .Value anymore after types moved is the only reason for the change.